README.rst

```diff
@@ -214,15 +214,34 @@ Supported features
 
```

Comment:
Can we squash all README changes together.
Also I think we should be inline with the release notes produced by TMEs
(I'll forward that to you)